metals/src/main/scala/scala/meta/internal/builds/BuildTools.scala

```diff
@@ -53,14 +53,14 @@ final class BuildTools(
       }
     }
   }
-  def isMill: Boolean = workspace.resolve("build.sc").isFile
+  private def isMill: Boolean = workspace.resolve("build.sc").isFile
   def isGradle: Boolean =
     workspace.resolve("build.gradle").isFile || workspace
       .resolve("build.gradle.kts")
       .isFile
   def isMaven: Boolean = workspace.resolve("pom.xml").isFile
   def isPants: Boolean = workspace.resolve("pants.ini").isFile
-  def isBazel: Boolean = workspace.resolve("WORKSPACE").isFile
+  private def isBazel: Boolean = workspace.resolve("WORKSPACE").isFile
```

Comment:
I would leave out private in this case - it's inconsistent with the other `is` methods.

Reply:
Yes, fixed.